packages/shared/src/components/post/smartPrompts/SmartPromptResponse.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useMemo } from 'react';
+import type { ReactElement } from 'react';
+import type { Post } from '../../../graphql/posts';
+import { RenderMarkdown } from '../../RenderMarkdown';
+import { SearchProgressBar } from '../../search';
+import Alert, { AlertType } from '../../widgets/Alert';
+import { isNullOrUndefined } from '../../../lib/func';
+import { labels } from '../../../lib';
+import { usePromptsQuery } from '../../../hooks/prompt/usePromptsQuery';
+import { useSmartPrompt } from '../../../hooks/prompt/useSmartPrompt';
+
+type SmartPromptResponseProps = {
+  post: Post;
+  activePrompt: string;
+};
+
+export const SmartPromptResponse = ({
+  post,
+  activePrompt,
+}: SmartPromptResponseProps): ReactElement => {
+  const { data: prompts } = usePromptsQuery();
+  const prompt = useMemo(
+    () => prompts?.find((p) => p.id === activePrompt),
+    [activePrompt, prompts],
+  );
+
+  const { executePrompt, data, isPending } = useSmartPrompt({ post, prompt });
+
+  useEffect(() => {
+    if (!prompt.prompt || data) {
+      return;
+    }
+
+    executePrompt(prompt.prompt + new Date().getTime());
```

Comment:
```suggestion
    executePrompt(prompt.prompt);
```
Keeping this in for a while, it's so that we can force trigger a new query from Magni